ccdproc/tests/test_image_collection.py

```diff
@@ -433,7 +429,7 @@ def test_dir_with_no_fits_files(self, tmpdir):
         empty_dir = tmpdir.mkdtemp()
         some_file = empty_dir.join('some_file.txt')
         some_file.dump('words')
-        with catch_warnings() as w:
+        with pytest.warns(Warning) as w:
```

Comment:
Not sure what the original intent was.

Reply:
An empty directory (or at least one with no FITS files) is supposed to issue a warning.